application/src/main/java/org/togetherjava/tjbot/moderation/ModAuditLogWriter.java

```diff
@@ -83,6 +83,7 @@ public void write(@NotNull String title, @NotNull String description, @NotNull U
      * this method will return an empty optional, and a warning message will be written.
      *
      * @param guild the guild to look for the channel in
+     * @return the channel used for moderation audit logs, if present
```

Comment:
> the channel used for moderation audit logs, if present

Sounds like

> the channel, or null when not present.

But actually it returns an Optional of the channel, maybe it has to do with the way I read

Reply:
Well, I guess the return **type** is obvious from the signature. I usually write it the same way as if it would be null, since the Javadoc operates a level higher than Javas typesystem. Like, it doesnt care how its represented 🤷

Reply:
Disagree with that, don't like it.
I'd rather read 

```java
@return An {@link Optional} of {@link TextChannel} which is the channel used for moderation audit logs.
```

Reply:
You are documenting the method, so it should reflect the method?

I don't understand this and sadly never noticed it, but if no-one else cares we can keep it like this ig